bundles/org.openhab.binding.chromecast/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -244,7 +244,8 @@
 		<item-type>DateTime</item-type>
 		<label>Broadcast Date</label>
 		<description>The broadcast date of the currently playing media</description>
-		<state readOnly="true"/>
+		<category>time</category>
```

Comment:
The first letter is always in uppercase.
I don't know if this is a problem with a lowercase first letter.
Can you update to be consistent with all other bindings ?
And by the way, `Date` looks more appropriate.

Reply:
The category is used to pick the default icon rendered in UIs. So I do not see a reason to write it in camel case.

We do not have an icon for "date". What I can offer is "calendar".

Reply:
I have a doubt, I don"t find `Date` in the documentation
https://www.openhab.org/docs/concepts/categories.html#channel-categories
But `Date` as channel category is used for example in the NTP binding.

Reply:
> > The first letter is always in uppercase.

> So I do not see a reason to write it in camel case.

Hm I see you are right. I will change it.

Reply:
Regarding `Date` as channel category, we have to fix the MI IO and the NTP bindings:
```
$ find ./*/src -name "*.xml" -exec grep -H "<category>Date" {} \;
./org.openhab.binding.miio/src/main/resources/OH-INF/thing/vacuumThing.xml:             <category>Date</category>
./org.openhab.binding.miio/src/main/resources/OH-INF/thing/vacuumThing.xml:             <category>Date</category>
./org.openhab.binding.ntp/src/main/resources/OH-INF/thing/thing-types.xml:              <category>Date</category>
./org.openhab.binding.ntp/src/main/resources/OH-INF/thing/thing-types.xml:              <category>Date</category>
```

Reply:
I changed the category to "Calendar".